### PR TITLE
Make preflight look pretty like PostHog

### PR DESCRIPTION
**Pull request by marcushyett-ph**
## Changes

A user commented this morning asking if their instance was broken when looking at the preflight screen, this inspired me to make some quick changes. This probably isn't the most elegant preflight page - but its a bit prettier and holiday-hackathon worthy.

## How did you test this code?
Before:

image

After:

image

Manual testing: Validated all of the functions of pre-flights continue to operate as before:

- Experimental instance
- Live instance
- Ask Support

### CODE REVIEW

File: frontend/src/scenes/PreflightCheck/index.tsx [hunk 1]
```diff
@@ -150,26 +153,46 @@ export function PreflightCheck(): JSX.Element {
...
+            <Row style={{ display: 'flex', justifyContent: 'center', paddingBottom: 16 }}>
+                <PageHeader title="Lets get started..." />
+            </Row>
+            <Row style={{ display: 'flex', justifyContent: 'center' }}>
+                <div style={{ width: 960 }}>
+                    <Steps current={0}>
+                        <Step title="Preflight check" subTitle="1 min" description="Prepare your instance" />
+                        <Step
+                            title="Event capture"
+                            subTitle="15 mins"
```
Comment (guidoiaquinti) [inline frontend/src/scenes/PreflightCheck/index.tsx:176]: What about 10? 😉

**Review by guidoiaquinti — APPROVED** [review]

File: frontend/src/scenes/PreflightCheck/index.tsx (hunk 1 above)
Comment (marcushyett-ph) [reply frontend/src/scenes/PreflightCheck/index.tsx:176]: I guess this is mostly out of our control - depends how difficult their app is to instrument. I felt 15mins was long enough to instrument something on a more complex [maybe mobile] app (so we don't make it sound easier than it is) but short enough not to put people off forging ahead.

**Comment by paolodamico** [comment]
This is great! Clear improvement. FYI we have [this design](https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=3836%3A29834) from Chris ready to be implemented. It's on my to-do for after the break but just in case you want to use it as inspiration.

**Comment by marcushyett-ph** [comment]
Wow the new design looks great - I'll leave this as a temporary fix until this full improvement gets prioritized

**Pull request merged**